test/e2e/resources/pipeline-maven-jib.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  labels:
+    name: task-jib-maven
+  name: task-jib-maven
+spec:
+  workspaces:
+    - name: source
+      optional: false
+    - name: dockerconfig
+      optional: true
+
+  params:
+    - name: URL
+      type: string
+    - name: REVISION
+      type: string
+    - name: VERBOSE
+      type: string
+    - name: IMAGE
+      type: string
+
+  tasks:
+    - name: git-clone
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/openshift-pipelines/tektoncd-catalog
+          - name: revision
+            value: p
+          - name: pathInRepo
+            value: tasks/task-git-clone/0.3.0/task-git-clone.yaml
+      workspaces:
+        - name: output
+          workspace: source
+      params:
+        - name: URL
+          value: "$(params.URL)"
+        - name: REVISION
+          value: "$(params.REVISION)"
+        - name: SUBMODULES
+          value: "false"
+        - name: VERBOSE
+          value: "$(params.VERBOSE)"
+        - name: SUBDIRECTORY
+          value: $(context.pipelineRun.name)
+    - name: maven-clean
+      taskRef:
+        name: maven
```

Comment:
This also we need to use cluster resolver isn't it ??

Reply:
we can yes, but not needed necessarily since make install, applys all task in current namespace so it can be directly accessed.